src/e-cherry-pick.js

```diff
@@ -131,143 +138,100 @@ program
       patchUrlStr = targetBranch;
       targetBranch = tmp;
     }
-    const octokit = new Octokit({
-      auth: await getGitHubAuthToken(['repo']),
-    });
-    try {
-      const {
-        data: { permissions },
-      } = await octokit.repos.get({
-        owner: 'electron',
-        repo: 'electron',
-      });
-      if (!permissions?.push) {
-        fatal(
-          'The supplied $GITHUB_TOKEN does not have write access to electron/electron - exiting',
-        );
-      }
 
+    const tmp = fs.mkdtempSync(path.join(os.tmpdir(), 'electron-tmp'));
+    const electronPath = path.join(tmp, 'electron');
+
+    const { remotes } = current();
+
+    try {
       const { patchDirName, shortCommit, patch, bugNumber, cve } = await getPatchDetailsFromURL(
         patchUrlStr,
         security,
       );
 
-      const patchName = `cherry-pick-${shortCommit}.patch`;
+      const patchName = `cherry-pick-${shortCommit}1.patch`;
```

Comment:
I assume this was left in for testing purposes? Can clean it up?

Reply:
yes, sorry!